 try:
        subprocess.run(command)
    except:
        print("error", j)
        continue
